### ISSUE TITLE
Server MCPE Offline

### ISSUE THREAD

**Issue by NycuRO** [JungleTree/JungleTree#36]
MCPE Server was offline.

**Comment by ojbr** [comment]
Yeah, my apologies. I am aware of it because we had a server migration over Christmas. The next 2 weeks I'm super busy on trying to get the codebase in such a state where we can put a newer version online. I'll update it here when that happens
